template.tpl

```diff
@@ -1230,10 +1241,37 @@ const WRAPPER_NAMESPACE = '_amplitude';
 
 // Print a log message and set the tag to failed state
 const fail = msg => {
-  log(LOG_PREFIX + 'Error: ' + msg);
   return data.gtmOnFailure();
 };
 
+// Merge two Objects
+const mergeObject = (basedObject, overwriteObject) => {
+  if (!basedObject || Object.keys(basedObject).length == 0) return overwriteObject;
+  if (!basedObject || Object.keys(overwriteObject).length == 0) return basedObject;
```

Comment:
Same here. What if overwriteObject is null? Will Object.keys(null) throw an exception?

Reply:
No, If the input value is not an object, it will be coerced to an object.
https://developers.google.com/tag-platform/tag-manager/templates/api#object_2